ocaml/xapi/cli_operations.ml

```diff
@@ -4649,3 +4649,31 @@ let lvhd_enable_thin_provisioning printer rpc session_id params =
 	let allocation_quantum = Record_util.bytes_of_string "allocation-quantum" (List.assoc "allocation-quantum" params) in
 	let ref = Client.SR.get_by_uuid rpc session_id uuid in
 	Client.LVHD.enable_thin_provisioning rpc session_id ref initial_allocation allocation_quantum
+
+module PVS_farm = struct
+	let get name params =
+		try
+			List.assoc name params
+		with Not_found ->
+			raise (Failure (Printf.sprintf "can't find parameter %s (%s)"
+				name __LOC__))
+
+	let print printer params names =
+		names
+		|> List.map (fun name -> get name params)
+		|> fun ps -> (Cli_printer.PList ps)
+		|> printer
+
+	let introduce printer rpc session_id params =
+		print printer params ["name"; "sr"]
```

Comment:
The `Client.PVS_farm` functions should exist now that you have created them in the datamodel, so you could just complete the CLI implementation already.